src/Form/FilePersisterServiceSettingsForm.php

```diff
@@ -47,7 +47,12 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#required' => TRUE
 
     ];
-
+    $form['file_path'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Relative Path for Persisting Files'),
+      '#description' => $this->t('Path relative to the root of the storage scheme selected above where the hashed directories used to store persisted managed files will be created. Do not include beginning or ending slashes. Default is "".'),
```

Comment:
Let's add a warning here. That says "changing this path after your ingests won't modify existing  Files"

Reply:
Ok, I added this note to both fields. Tell me if that's not correct!

Reply:
Thanks. Correct!